plugins/timesync.koplugin/main.lua

```diff
@@ -9,21 +9,28 @@ if C.getuid() ~= 0 then
     return { disabled = true, }
 end
 
+local function which(command)
+    local path = os.getenv("PATH") or ""
+    for p in path:gmatch("([^:]+)") do
+        p = p .. "/" .. command
+        if lfs.attributes(p) and os.execute("test -x " .. p) == 0 then
+            return p
+        end
+    end
+end
+
```

Comment:
This should be moved to `frontend/util.lua` so it can be used by others.

Reply:
Done